config/settings.yml

```diff
@@ -40,6 +40,10 @@ ssoe_eauth_cookie:
 session_cookie:
   secure: false
 
+lockbox:
+  master_key: "a798f118b400a1a7552597cc94f44c577919c76b3d26c1eed8ae0703f4240216"
+  #old_master_key: we will need this in the near future when rotating
```

Comment:
I plan to integrate the previous_key in a separate PR later on to avoid introducing multiple changes in a single PR

Reply:
I'd recommend "previous_key" instead of old key (saw it in the lockbox docs).